checks.d/wmi_check.py

```diff
@@ -109,7 +109,7 @@ def _extract_metrics(self, results, metrics, tag_by, wmi, tag_queries, constant_
                     # Special-case metric will just submit 1 for every value
                     # returned in the result.
                     val = 1
-                else:
+                elif getattr(res, wmi_property) is not None:
```

Comment:
```python
elif getattr(res, wmi_property):
```
is probably enough as we don't want to cast an empty string either.

Also, should we add a `log.warning` when we endup in this case ? It could help users or support understanding why some metrics are silenced.